### PR TITLE
fixes bad url file handling issue #525

### PR DESCRIPTION
**Pull request by DoomTaper**

### CODE REVIEW

File: framework/interface/templates/plugin_report.html [hunk 1]
```diff
@@ -94,7 +95,7 @@ <h4 id="header_badge_{{ test_code }}"></h4>
                                         <td> {{ poutput['start_time'] }} <br />{{ poutput['end_time'] }} </td>
                                         <td> {{ poutput['status'] }} </td>
                                         {% if poutput['output_path'] %}
-                                            <td><a href="/output_files/{{ poutput['output_path'] }}/" target="_blank" class="btn btn-primary">Browse</a></td>
+                                            <td><a href="/output_files//{{ quote_plus(poutput['output_path'], safe='/') }}/" target="_blank" class="btn btn-primary">Browse</a></td>
```
Comment (delta24) [inline framework/interface/templates/plugin_report.html]: Why the extra `/` ?

File: framework/interface/ui_handlers.py [hunk 2]
```diff
@@ -440,11 +441,12 @@ class FileRedirectHandler(custom_handlers.UIRequestHandler):
     SUPPORTED_METHODS = ('GET')
 
     def get(self, file_url):
+	file_url = file_url[1:]
```
Comment (delta24) [inline framework/interface/ui_handlers.py]: Intendation.

File: framework/interface/custom_handlers.py [hunk 3]
```diff
@@ -34,7 +35,7 @@ def get(self, path, include_body=True):
         This is an edited method of original class so that we can show
         directory listing and set correct Content-Type
         """
-        path = self.parse_url_path(path)
+	path = self.parse_url_path(path)
```
Comment (delta24) [inline framework/interface/custom_handlers.py]: Indentation.

File: framework/interface/templates/plugin_report.html (hunk 1 above)
Comment (DoomTaper) [inline framework/interface/templates/plugin_report.html]: By intercepting the request in burp suite I observed that if I dont put /(or any other character) then it is not making the correct GET request but I am not able reproduce it now :confused: .You check that @delta24  and if everything is fine make the appropriate changes.

File: framework/interface/ui_handlers.py (hunk 2 above)
Comment (DoomTaper) [inline framework/interface/ui_handlers.py]: It doesn't show any indentation error in my kali and also doesn't show any false indentation in my develop branch files. I don't why it is showing in the commit.

Comment (delta24) [inline framework/interface/ui_handlers.py]: :P
Check once more in any other editor.

Comment (DoomTaper) [inline framework/interface/ui_handlers.py]: I have already checked them on Gvim, Vim, gedit and leafpad, they were not showing any indentation problem but just now I checked it on sublime text 3 and it shows the indentation problem :P

Comment (delta24) [inline framework/interface/ui_handlers.py]: It probably means there was a mixed indentation (tabs and spaces). Always use spaces. :)

File: framework/interface/templates/plugin_report.html (hunk 1 above)
Comment (delta24) [inline framework/interface/templates/plugin_report.html]: Remove `file_url = file_url[1:]` and you will not need it.

Comment (DoomTaper) [inline framework/interface/templates/plugin_report.html]: Ya! it was due to that , my bad!! So we doesn't require it. thanks @delta24

File: framework/interface/ui_handlers.py (hunk 2 above)
Comment (DoomTaper) [inline framework/interface/ui_handlers.py]: thanks @delta24

**Comment by delta24** [comment]
Wrong approach. I am committing the correct fix in a few minutes.

**Pull request closed without merging**